### PR TITLE
Update LICENSE file

### PR DESCRIPTION
**Pull request by mariusvn**
Working on updating the license file. The license is already in 3 clause BSD. I wanna make sure we don't need to add/remove a license part.
The main point i'm not sure about is : should i add a new 3 clause BSD section for Maplibre and keep the original text saying that it has been in the m0nac0's hands ?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-flutter-maplibre-gl Copyright (c) 2021, m0nac0.
+flutter-maplibre-gl Copyright (c) 2023, MapLibre contributors.
```
Comment (wipfli) [inline LICENSE:1]:
```suggestion
flutter-maplibre-gl Copyright (c) 2023, MapLibre contributors.
flutter-maplibre-gl Copyright (c) 2021, m0nac0.
```

**Comment by wipfli** [comment]
Thanks for looking into this. I suggest to just add one line at the top that it is MapLibre contributors now.

**Comment by mariusvn** [comment]
Okay, I applied your suggestion

**Review by birkskyum — APPROVED** [review]

**Comment by m0nac0** [comment]
@mariusvn Thank you for looking into the "special files"!

**Pull request merged**